main.py

```diff
@@ -40,13 +40,13 @@ async def send_pages(self):
 client.help_command = MyHelpCommand()
 
 
-@client.command(name="getUTC")
+@client.command(name="getUTC", help="Gets the current UTC time.")
 async def getUTC(ctx) -> None:
     utc_time = await getUTCTime()
     await ctx.send(utc_time.strftime("%Y-%m-%d %H:%M:%S"))
 
 
-@client.command(name="fetch")
+@client.command(name="fetch", help="Fetch the latest Nadocast images. \n Usage: $fetch <param> \n Allowed params: sig, tor, wind, hail")
```

Comment:
No clue if the user would assume only one parameter is expected. Maybe the word should be re-worded? 
 ```suggestion
@client.command(name="fetch", help="Fetch the latest Nadocast images. \n Usage: $fetch <params> \n Allowed params: sig, tor, wind, hail\n Examples: `$fetch tor`, `$fetch sig tor`")
```

Reply:
Oh cool didn't know you can just commit the suggestion instead of doing it manually from my end lol

Reply:
Yeah I found out how to use suggestions lol